pages/issues.py

```diff
@@ -0,0 +1,61 @@
+from enum import Enum
+from selenium import webdriver
+from selenium.webdriver.common.by import By
+
+
+class IssueStatuses(Enum):
+    """ Enum of types for Issue Statuses. """
+    OPEN = '0'
+    IN_PROGRESS = '1'
+    RESOLVE = '2'
+
+
+class IssuesPage:
+    """ Locators and methods for Issues page. """
+
+    VIEW_BTN_CSS = "tr:nth-child({})  a > button"
+    FROM_DATA_INP_CSS = "div:nth-child(1) > div > div > input"
+    TO_DATA_INP_CSS = "div:nth-child(2) > div > div > input"
+    CHECK_CSS = "input[name='status[{}]']"
+    RESET_BTN_XPATH = "//span[contains(text(),'Reset')]"
+    SEARCH_BTN_XPATH = "//span[contains(text(),'Search')]"
+
+    def __init__(self):
+        self.driver = webdriver.Chrome()
+
+    def click_view(self, view_number):
+        """
+         Method for click on a needed view button.
+
+                Paremeters:
+                    view_number (int): A decimal integer
+
+        """
+        element = self.driver.find_element(By.CSS_SELECTOR, self.VIEW_BTN_CSS.format(view_number + 1), view_number)
+        element.click()
+
+    """
+        There should be a code for entering the date in the datepicker but I cannot fill it with data.
+        Please, help me...
+    """
+
+    def click_issue_status_filter(self, issue_status):
+        """
+            Method for click on a needed issue status checkbox.
+        """
+        element = self.driver.find_element(By.CSS_SELECTOR, self.CHECK_CSS.format(issue_status.value))
+        element.click()
```

Comment:
self.CHECK_CSS = ".checkbox > label"

```suggestion
    def click_issue_status_filter(self, issue_status):
        """
            Method for click on issue status checkbox depending on text value.
        """
        elements = self.driver.find_elements(By.CSS_SELECTOR, self.CHECK_CSS)
        for element in elements:
            if issue_status == element.text:
                element.click()
                return
```
And no need for IssueStatuses class :smile:

Reply:
done